environments/site/tofu/node_group/variables.tf

```diff
@@ -192,3 +192,12 @@ variable "allowed_keys" {
     type = list
     # don't provide a default here as allowed keys may depend on module use
 }
+
+variable "config_drive" {
+    description = <<-EOT
```

Comment:
We/I haven't been consistent on this, but if its just passed through I don't think you need to add the description, so might be easier to update than to make equivalent changes for below comment. Whichever is fine.